### ISSUE TITLE
Accessibility Review of Focus State

### ISSUE THREAD

**Issue by beparticular** [18F/cms-hitech-apd#2191]
In testing we saw that there is some inconsistency in where the focus is when a page is loaded. Sometimes it remains on the continue/previous button, side nav, etc. 

This could be an issue for our upcoming accessibility review for ATO as the focus on the page should be located on the element that makes most sense for starting the page for all users, including screen readers. 

### This task is done when...
 - [ ] a review of each page in the application has been completed, noting where the focus needs to be when the page is loaded
 - [ ] we have a list of pages where the focus needs to be changed that will go into a development ticket
 - [ ]  note if autofocus is being implemented and where
 - [ ] Have a brown bag on screen reader review on examples (optional)

**Comment by beparticular** [comment]
Tools that may help us with accessibility reviews:
https://wave.webaim.org/
https://a11yproject.com/checklist/

**Comment by beparticular** [comment]
Status: I have a meeting scheduled with Heather this Friday to learn about past accessibility work

**Comment by beparticular** [comment]
Recap from the meeting with Heather - 

- It's possible the issue we're seeing with focus states has something to do with the way React reloads pages and how we currently render pages. If the page doesn't reload, the button might stay the point of focus. 

- For accessibility review, 
1) look to see if the button has a tab index 
2) see if one can successfully navigate those pages via keyboard, log where it goes around focus states

**Comment by beparticular** [comment]
A more comprehensive task has been created #2358 for a full accessibility audit of the system, vs just the focus states. Closing this one so we don't have duplicates.
